module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php

```diff
@@ -64,17 +75,18 @@ public function process(FileExportChannel $channel, AbstractProduct $product): E
         }
     }
 
-    /**
-     * @param array $attributes
-     */
-    private function getLanguage(AbstractProduct $product, Language $language, array $attributes): LanguageData
-    {
+    private function getLanguage(
+        AbstractProduct $product,
+        Language $language,
+        array $attributes,
+        Template $template
+    ): LanguageData {
         $result = new LanguageData();
         $result->set('_id', $product->getId()->getValue());
         $result->set('_sku', $product->getSku()->getValue());
         $result->set('_type', $product->getType());
         $result->set('_language', $language->getCode());
-        $result->set('_template', $product->getTemplateId()->getValue());
+        $result->set('_template', $template->getName());
```

Comment:
Why change from `id` to `name`?

Reply:
Basically via import new ids being created therefore those cant really be used